Dockerfile

```diff
@@ -23,5 +23,7 @@ RUN pip install --target="/install" /src
 FROM python:${PY_VERSION}-slim
 ARG PY_VERSION
 COPY --from=builder /install /usr/local/lib/python${PY_VERSION}/site-packages
-COPY gpt3discord.py /bin/gpt3discord
-CMD ["python3", "/bin/gpt3discord"]
+RUN mkdir -p /opt/gpt3discord/etc
+COPY gpt3discord.py /opt/gpt3discord/bin/
+COPY image_optimizer_pretext.txt conversation_starter_pretext.txt conversation_starter_pretext_minimal.txt /opt/gpt3discord/share/
```

Comment:
Do we need to test for openers first and mkdir it if it doesn't exist? Or will having 
`COPY openers /opt/gpt3discord/share/openers` after the image optimizer pretext copy lines just work?

Reply:
Copy makes a new folder and an existing one would never exist since we're copying to a newly created image

![image](https://user-images.githubusercontent.com/15952522/211480698-c76c25f7-ac26-4898-ad72-84c672d0ea5a.png)